tests/t068_filter_time_A.py

```diff
@@ -15,13 +15,6 @@ def __init__(self):
    2.107 ms [23157] | } /* main */
 """)
 
-    def build(self, name, cflags='', ldflags=''):
-        # cygprof doesn't support return value now
-        if cflags.find('-finstrument-functions') >= 0:
-            return TestBase.TEST_SKIP
-
-        return TestBase.build(self, name, cflags, ldflags)
-
     def setup(self):
```

Comment:
Actually, this file is not related to description of this commit. 
I found this test passed without that function. 
I'll exclude this file if you want. please let me know.
Thank you.

Reply:
Yes, it can be in another patch.

Reply:
ok. I divided it.